gabrielaEnache/src/main/java/oop/store/AbstractUser.java

```diff
@@ -0,0 +1,54 @@
+package oop.store;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public abstract class AbstractUser {
+    protected final long id;
```

Comment:
Take a look on the above comment regarding access modifiers, here you can use private ones.

Reply:
Fixed